ginivision/src/main/java/net/gini/android/vision/internal/camera/api/Util.java

```diff
@@ -14,15 +15,45 @@
 public final class Util {
```

Comment:
This `Util` class is very Aspect Ratio and sizes orientated. Perhaps it's not really utils, but `SizeUtils`, `AspectRatios` or `ResolutionSelection`?